inlong-manager/manager-pojo/src/main/java/org/apache/inlong/manager/pojo/cluster/ClusterTagResponse.java

```diff
@@ -50,6 +50,9 @@ public class ClusterTagResponse {
     @ApiModelProperty(value = "Extended params")
     private String extParams;
 
+    @ApiModelProperty(value = "The compression type used for dataproxy and sort side data transmission to reduce the network IO overhead")
+    private String inlongCompressType;
```

Comment:
```suggestion
    private String ompressType;
```

Reply:
This name is used to distinguish it from other compress types, such as hive's compress type.